src/App.css

```diff
@@ -32,63 +32,7 @@
     transform: rotate(360deg);
   }
 }
-.login {
-  display: block;
-  background-color: white;
-  width: 350px;
-  height: 400px;
-  margin: 10px;
-}
-.inputs {
-  width: 300px !important;
-  margin-left: 20px;
```

Comment:
Is removing this file required to fix the login ui?

Reply:
It was feedback about putting all the css that affects the index page and dashboard in a new style folder

Reply:
That is clean up or refactoring. It has nothing to do with fixing the login url in the sense that even if you did not move it around, the login would still be fixed by the other changes. Hence qualifying to require its separate ticket. Not so? Unless moving it in a new style folder is what fixed the login. 😊

Reply:
So what should I do because, I already implemented all the feedback from people as fixing the login UI

Reply:
Revert those changes that have nothing to do with the actual fix. Then create a separate ticket and pull request for them.